Eatery/Controllers/BRB/BRBAccountManager.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  AccountManager.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 10/24/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import WebKit
+
+struct HistoryEntry {
+
+    var description: String = ""
+    var timestamp: String = ""
+
+}
+
+struct AccountBalance {
+
+    var brbs: String = ""
+    var cityBucks: String = ""
+    var laundry: String = ""
+    var swipes: String = "0"
+
+}
+
+enum Stages {
+
+    case loginScreen
+    case transition
+    case loginFailed
+    case finished(sessionId: String)
+
+}
+
+//MARK: -
+//MARK: Account Settings
+
+typealias LoginInfo = (netid: String, password: String)
+
+private enum BRBAccountSettings {
+    
+    static func saveToKeychain(loginInfo: LoginInfo) {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = loginInfo.netid as AnyObject
+        keychain["password"] = loginInfo.password as AnyObject
+    }
+    
+    static func removeKeychainLoginInfo() {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        keychain["netid"] = nil
+        keychain["password"] = nil
+    }
+    
+    static func loadFromKeychain() -> LoginInfo? {
+        let keychain = KeychainItemWrapper(identifier: "netid", accessGroup: nil)
+        guard let netid = keychain["netid"] as? String, let password = keychain["password"] as? String else {
+            return nil
+        }
+        return (netid: netid, password: password)
+    }
+    
+}
+
+private let loginURLString = "https://get.cbord.com/cornell/full/login.php?mobileapp=1"
+private let maxTrials = 3
+private let trialDelay = 500
+
+//MARK: -
+//MARK: Account Manager
+
+protocol BRBAccountManagerDelegate {
+    func failedToGetAccount(with error: String)
```

Comment:
Same thing with these two delegates as well. 

I know these were in the Eatery codebase like this before, but when we touch something we should always try and make it cleaner than we found it.